modules/statics/src/coins.ts

```diff
@@ -9123,6 +9123,24 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.LQID,
     AccountCoin.DEFAULT_FEATURES
   ),
+  solToken(
+    'd82f66bf-e921-44e4-a9b0-462eda94a9ad',
+    'sol:vi',
+    'VI',
+    6,
+    '7zBWymxbZt7PVHQzfi3i85frc1YRiQc23K7bh3gos8ZC',
+    UnderlyingAsset.VI,
+    AccountCoin.DEFAULT_FEATURES
+  ),
+  solToken(
+    '5f98c609-de50-41b4-8f82-c59e5273166e',
+    'sol:ftt-sollet',
```

Comment:
```suggestion
    'sol:softt',
```
Per ticket and market symbol specification.

Reply:
Fixed per ticket updates.

Reply:
Updated the ticket description to match the name used here